kubernetes/airflow/values.yaml

```
@@ -552,7 +552,7 @@ workers:
   # extraInitContainers: []
   extraInitContainers:
     - name: fix-volume-logs-permissions
-      image: busybox
+      image:  "{{ .Values.global.imageRegistry | default \"ghcr.io\" }}/svtechnmaa/busybox:1.33"
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)